src/claimtrie.cpp

```diff
@@ -238,11 +198,9 @@ void CClaimTrie::clear(CClaimTrieNode* current)
 bool CClaimTrie::haveClaim(const std::string& name, const COutPoint& outPoint) const
 {
     const CClaimTrieNode* current = &root;
-    for (std::string::const_iterator itname = name.begin(); itname != name.end(); ++itname)
-    {
+    for (std::string::const_iterator itname = name.begin(); itname != name.end(); ++itname) {
         nodeMapType::const_iterator itchildren = current->children.find(*itname);
-        if (itchildren == current->children.end())
-            return false;
+        if (itchildren == current->children.end()) return false;
```

Comment:
If this style change (and others like it) isn't pulled in from clang-format, remove it. (Please apply to other cases as well -- alternatively, just run clang-format and rebase and comment that that's why it's here.)